### PR TITLE
Feature/day 5

### PR DESCRIPTION
**Pull request by anantmishra-successive**
Summary :
day5Summary

### CODE REVIEW

File: Day5/que5.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Write a program flattenObject that takes an object with nested properties and returns a flat object with the nested properties transformed into dot-separated properties. The function should handle nested objects and arrays.
```
Comment (Kartik-ST) [inline Day5/que5.js:1]: @anantmishra-successive the o/p of this ques should have a flat object with nested keys dot separated but not starting with dot. 
Secondly, you are not supposed to flatten array.

Comment (anantmishra-successive) [reply Day5/que5.js:1]: Sir
Resolved

Comment (anantmishra-successive) [reply Day5/que5.js:1]: updateday5

Comment (Kartik-ST) [reply Day5/que5.js:1]: @anantmishra-successive 
1.You have added extra logic. Do it by manipulating same old logic.
2. And DO NOT resolve the comments by yourself.
3. I also wrote that you are not supposed to flatten the array. You haven't yet changed that.

Comment (anantmishra-successive) [reply Day5/que5.js:1]: day5final

**Pull request merged**